src/components/DashboardResultIndicatorsSection.vue

```diff
@@ -294,18 +294,22 @@ export default {
     },
     formatResultIndicatorValue(value) {
       const resultIndicator = this.resultIndicators[this.activeTab];
-      if (resultIndicator && value) {
-        if (resultIndicator.type === 'users') {
-          return numberLocale.format(',')(value);
-        }
-        return kpiTypes[resultIndicator.type].formatValue(value);
-      }
-      return null;
+      return resultIndicator && value
+        ? kpiTypes[resultIndicator.type].formatValue(value)
+        : null;
     },
     download(value) {
       const filename = this.resultIndicators[this.activeTab].name;
       const svgData = this.$refs.progressGraphSvg;
 
+      // Hide certain elements from the graph before exporting
+      const hiddenElements = [svgData.querySelector('.indicators')].filter(
+        (el) => el
+      );
+      hiddenElements.forEach((el) => {
+        el.style.opacity = 0;
+      });
```

Comment:
Dette skjønte jeg ikke helt. Finnes det bare ett `.indicators`-element? I så fall kan man vel bare sette `style` på elementet som returneres fra `querySelector` direkte?

Reply:
Ja, eller så må du vel uansett bruke `querySelectorAll` for å hente ut alle? Men enig at det er bedre å vente til det eventuelt blir tilfelle.